Yoast/javascript
# Add exception check for short French words that are not stemmed but should be

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Stems French words that are considered too short to be stemmed according to the stemming rules, but that should nevertheless be stemmed.
* [Yoast SEO Premium] Improves word form recognition of short French words (e.g. _ours_ - _our_; _âmes_ - _âme_)

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all current specs pass, and that the coverage of the stemmer is 100 %
* Add more examples of words from the exception lists (shortWordsAndStems in the morphology data file) to the specs. For the words on the canTakeExtraSuffixS sub-list, also try adding -s at the end of the word (e.g. both _île_ and _îles_ should be stemmed to _îl_).
* In French Premium: add forms of the words from the exception lists to the focus keyphrase and to the title/text/meta description and make sure they are recognized.

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-403
